src/server/journal/cmd_serializer.cc

```diff
@@ -213,7 +214,7 @@ void CmdSerializer::SerializeRestore(string_view key, const PrimeValue& pk, cons
   args.push_back(expire_str);
 
   io::StringSink value_dump_sink;
-  SerializerBase::DumpObject(pv, &value_dump_sink, true);
+  SerializerBase::DumpObject(serializer_.get(), pv, &value_dump_sink);
```

Comment:
please pass `ignore_crc=false` explicitly and in generally, I am really against using default arguments in such cases.

Reply:
ah, I see you changed the interface